xml/installation-installation-ironic-ironic_provisioning.xml

```diff
@@ -571,8 +571,7 @@ ssh ubuntu@192.3.15.14 -i ironic_kp.pem
    <link xlink:href="http://docs.openstack.org/developer/diskimage-builder/user_guide/installation.html"/>.
   </para>
  </section>
- <!-- FIXME: no SUSE instructions here - sknorr, 2018-01-03 -->
```

Comment:
Is this issue now solved or is it a non-issue at this point?
The rest of this commit makes sense -- thanks for fixing the section structure!